index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- Link CSS -->
+	<link rel="stylesheet" href="./styles/style.css">
+	<!-- Font Link -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+	<title>Document</title>
+</head>
+
+<body>
+
+	<header>
+		<div class="nav-bar">
+			<a href="#">Hernán Zamora</a>
+			<div class="menu">
+				<div class="bar"></div>
+				<div class="bar"></div>
+				<div class="bar"></div>
+			</div>
```

Comment:
- [ ] I think it will be better if you wrap your hamburger in a `nav` tag for semantic reasons. Your `nav` tags will contain navigation elements and `nav` tags are usually used to wrap navigation items, so it is good you wrap your hamburger in a nav tag instead of a div